### ISSUE TITLE
Stryker hangs for long time once ng test started

### ISSUE THREAD

**Issue by SathyaNagaraj** [stryker-mutator/stryker-js#4459]
**Question**
Nothing is happening once ng test is started. Its idle for long time. I could not see any warning or error on the screen. 

![image](https://github.com/stryker-mutator/stryker-js/assets/16255724/c9cd5123-6adf-4894-9d4e-3f1253341b97)

**Stryker environment**

```
    "karma": "6.4.2",
    "karma-chrome-launcher": "3.1.0",
    "karma-coverage": "2.0.3",
    "karma-html-reporter": "0.2.7",
    "karma-jasmine": "5.1.0",
    "karma-jasmine-html-reporter": "2.0.0",
    "karma-spec-reporter": "0.0.32",

    "@stryker-mutator/karma-runner": "7.1.1",
    "@stryker-mutator/typescript-checker": "7.1.1",
```

I am using   "@angular/cli": "~15.2.0",

I could see mutated files in sandbox. 

![image](https://github.com/stryker-mutator/stryker-js/assets/16255724/6e040f2f-e52d-48ee-85fa-1c8d569c46f1)

![image](https://github.com/stryker-mutator/stryker-js/assets/16255724/62d210e4-f1f9-4670-9120-f8209fdc02b3)

I could see blank screen ,once ng test started

![image](https://github.com/stryker-mutator/stryker-js/assets/16255724/4bcc5020-c729-4d19-a4ac-395c6b1b54e2)

**Comment by saranyanandagopal** [comment]
I am facing the same issue, how did you solve this?

**Comment by nicojs** [comment]
Could you try `--logLevel trace`? That way you can see the karma output.

**Comment by saranyanandagopal** [comment]
Yes, I tried that already but I am not seeing anything after the "ng test" command (screen shot below). I am able to run my tests outside of Stryker normally.

![image](https://github.com/user-attachments/assets/f9f43e75-5495-4d58-b20a-2d663b6e8705)

**Comment by saranyanandagopal** [comment]
Any other inputs? When I run the "ng test" command from command line separately on the sandbox directory , I can see that things are starting up.
